cypress/e2e/flow-builder/connections/creating-removing.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Creating a connection', () => {
```

Comment:
- [ ] it's not only about "Creating". Let's rename it to "Creating/removing connections"